tree/20_cfe_basics/cfengine/paths.cf

```diff
@@ -101,12 +104,14 @@ bundle common paths
     _have_bin_systemctl::
       "path[systemctl]"      string => "/bin/systemctl";
     !_have_bin_systemctl::
-      "path[systemctl]"      string => "/usr/bin/systemctl";
+      "path[systemctl]"      string => "/bin/systemctl";
```

Comment:
i'm pretty sure this is not what we want:
` "_have_bin_systemctl" expression => fileexists("/bin/systemctl");`

Reply:
Fixed and opened https://github.com/cfengine/masterfiles/pull/1004